UI-extra/style.css

```diff
@@ -185,16 +195,19 @@ p {
 }
 
 .global-button {
-    border: 1px solid #ffe701;
-    background-color: #ffe701;
-    color: #404040;
-    border-color: #e6b905;
+    width: 100%;
+    margin: 10px 0;
+    font-size: 13px;
+    border-radius: 2px;
+    min-height: 40px;
 }
 
 .small-button {
     min-height: 35px;
     min-width: 95px;
-    border-radius: 2px;
+    border: 1px solid #e6b905;
```

Comment:
You can remove border, background-color and color from `.small-button`. Instead you can just add `.primary-button` to the button element so you are not restricting yourself that ALL small buttons are gonna look like primary button